charts/quine-enterprise/templates/basic-auth-secret.yaml

```
@@ -0,0 +1,9 @@
+{{- if .Values.basicAuth.enabled }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: credentials
```

```suggestion
  name: {{ include "quine-enterprise.fullname" . }}-credentials
```
It is important to include the name of the chart in resources like this so that multiple copies of the chart can be deployed in the same namespace without conflicting.

---

This is super smart and makes a lot of sense! I was reading the [Helm docs](https://helm.sh/docs/) and saw that there could be multiple [releases](https://helm.sh/docs/glossary/#release) of the same chart installed on the same cluster. This change enables that 🔥